Formula/xcode-kotlin.rb

```diff
@@ -0,0 +1,32 @@
+class XcodeKotlin < Formula
+  desc "Kotlin Native Xcode Plugin"
+  homepage "https://github.com/touchlab/xcode-kotlin"
+  url "https://github.com/touchlab/xcode-kotlin.git",
+    tag:      "1.2.0",
+    revision: "3af7e5d921c3756a3adc83ad504ac0db2a5643e8"
+  license "Apache-2.0"
+  head "https://github.com/touchlab/xcode-kotlin.git", branch: "main"
+  depends_on "gradle" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on :macos
+
+  def install
+    suffix = Hardware::CPU.arch == :x86_64 ? "X64" : "Arm64"
+    system "gradle", "--no-daemon", "linkReleaseExecutableMacos#{suffix}", "preparePlugin"
+    bin.install "build/bin/macos#{suffix}/releaseExecutable/xcode-kotlin.kexe" => "xcode-kotlin"
+    share.install Dir["build/share/*"]
+  end
+
+  test do
+    assert_match(/
+        (?:Plugin\snot\sinstalled.\sRun\s'xcode-kotlin\sinstall'\sto\sinstall\sit.)?\s*
+
+        Installed\splugin\sversion:\s*(?:(?:\d+)\.(?:\d+)\.(?:\d+)|none)\s*
+        Bundled\splugin\sversion:\s*(?:\d+)\.(?:\d+)\.(?:\d+)\s*
+
+        Language\sspec\sinstalled:\s*(?:Yes|No)\s*
+        LLDB\sinit\sinstalled:\s*(?:Yes|No)\s*
+        LLDB\sXcode\sinit\ssources\smain\sLLDB\sinit:\s*(?:Yes|No)\s*
+      /x, shell_output(bin/"xcode-kotlin info --only"))
```

Comment:
This will be easier to read:
```suggestion
    output = shell_output(bin/"xcode-kotlin info --only")
    assert_match " Bundled plugin version: #{version}", output
    assert_match /Installed plugin version:\s*(?:(?:\d+)\.(?:\d+)\.(?:\d+)|none)/, output    
    assert_match /Language spec installed:\s*(?:Yes|No)/, output
    assert_match /LLDB init installed:\s*(?:Yes|No)/, output
    assert_match /LLDB Xcode init sources main LLDB init:\s*(?:Yes|No)/, output
```